engine/class_modules/apl/apl_monk.cpp

```diff
@@ -238,9 +238,6 @@ void mistweaver( player_t *p )
   action_priority_list_t *def     = p->get_action_priority_list( "default" );
   action_priority_list_t *racials = p->get_action_priority_list( "race_actions" );
 
-  pre->add_action( "flask" );
-  pre->add_action( "food" );
-  pre->add_action( "augmentation" );
```

Comment:
It's standard for flask/food/aug to be included in default APLs.

Reply:
I got a warning saying
```
Flask Action has been depreciated and is no longer needed in the Precombat APL
Food Action has been depreciated and is no longer needed in the Precombat APL
Augmentation Rune Action has been depreciated and is no longer needed in the Precombat APL
```

Reply:
Oop, I forgot about #9781 being a thing.